### DISCUSSION TITLE
Additions to vimtex config

### DISCUSSION THREAD

**Discussion by firubat** [LazyVim/LazyVim#2516] [Ideas]
[vimtex](https://github.com/lervag/vimtex) has a few default keybindings that start with `t` (for `toggle`). This conflicts with the `t` movement keybinding. I suggest adding different keybindings for these. Here is my suggestion, based on the fact that vimtex has keybindings such as `cse` (change surrounding environment), `csc` ("change surrounding command") etc. and it seems (from which-key) that `cx...` is not bound to anything. Hope it doesn't conflict with anything.
```
vim.keymap.set({ "n" }, "cxe", "<plug>(vimtex-env-toggle-star)")
vim.keymap.set({ "n" }, "cxc", "<plug>(vimtex-cmd-toggle-star)")
vim.keymap.set({ "n" }, "cxf", "<plug>(vimtex-cmd-toggle-frac)")
vim.keymap.set({ "n" }, "cxm", "<plug>(vimtex-env-toggle-math)")
vim.keymap.set({ "n", "x" }, "cxd", "<plug>(vimtex-delim-toggle-modifier)")
vim.keymap.set({ "n", "x" }, "cxD", "<plug>(vimtex-delim-toggle-modifier-reverse)")
```
I also like to add these:
```
vim.keymap.set({ "n" }, "dsm", "<plug>(vimtex-env-delete-math)")
vim.keymap.set({ "n" }, "csm", "<plug>(vimtex-env-change-math)")
```
Which are mapped by defalt to `ds$` and `cs$` which are less convenient.

Another feature I like is adding paring for `$$` by adding this to the filetype configuration for tex and bib:
```
MiniPairs.map_buf(0, "i", "$", { action = "closeopen", pair = "$$" })
```

p.s. 
I put it here as "Idea" because I wasn't sure if it should be added as an issue for feature request - tell me if so and I'll add it.

**Comment by 108anup** [comment]
Another alternative is to use a prefix of maplocalleader, e.g., '\'
